src/modules/User/components/FavoriteRooms/FavoriteRooms.tsx

```diff
@@ -34,12 +29,11 @@ export default function FavoriteRoom() {
     startDate?: string,
     endDate?: string
   ) => {
-   
     navigate(`/dashboard/room-details/${roomId}`, {
       state: {
-        capacity: capacity || 2, 
-        startDate: startDate || format(new Date(), "yyyy-MM-dd"), 
-        endDate: endDate || format(new Date(), "yyyy-MM-dd"), 
+        capacity: capacity || 2,
```

Comment:
We can handle this params using searchParms 
`  navigate(`/dashboard/room-details/${roomId}`?startDate=${startDate}&endDate=${endDate}&capacity=${2}`